client/ipa-rmkeytab.c

```diff
@@ -111,6 +111,9 @@ remove_principal(krb5_context context, krb5_keytab ktid, const char *principal,
 
 done:
 
+    if (entry.principal)
+        krb5_free_principal(context, entry.principal);
```

Comment:
`krb5_free_principal` does not set entry.principal to NULL, should we?

Reply:
`entry` is a static local object which is invisible beyond this function. No need to NULL here.